.github/workflows/create-release.yml

```
@@ -60,27 +63,28 @@ jobs:
           name: hermetic-driver-${{ matrix.target }}
           path: target/${{ matrix.target }}/release/hermetic-driver
 
-  # release:
-  #   needs: [build-binaries]
-  #   runs-on: ubuntu-latest
-  #   env:
-  #     CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_TOKEN }}
-  #     GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #   steps:
-      # - uses: actions/checkout@v3
-      #   with:
-      #     fetch-depth: 0
-      # - uses: actions/download-artifact@v3
-      #   with:
-      #     path: artifacts
-      # - name: check artifacts
-      #   run: |
-      #     ls artifacts/**/*
-      # - name: Install Rust toolchain
-      #   uses: dtolnay/rust-toolchain@stable
-      # - id: release
-      #   run: |
-      #     export TAG=$(cargo metadata --format-version=1 --no-deps | jq '.packages[0].version' | tr -d '"')
-      #     echo "Releasing "$TAG
-      #     gh release create "v${TAG}" -n "Release of ${TAG}" -t "v${TAG}" --latest artifacts/**/*.tar.gz
\ No newline at end of file
+  release:
+    needs: [build-binaries]
+    runs-on: ubuntu-latest
+    env:
+      # CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_TOKEN }}
+      # GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/download-artifact@v3
+        with:
+          path: artifacts
+      - name: check artifacts
+        run: |
+          ls artifacts/**/*
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - id: release
+        run: |
+          export TAG=$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[0].version' )
+          echo "Releasing "$TAG
+          cp artifacts/hermetic-driver-x86_64-unknown-linux-gnu/hermetic-driver artifacts/hermetic-driver-x86_64-linux
+          gh release create "v${TAG}" -n "Release of ${TAG}" -t "v${TAG}" --latest artifacts/hermetic-driver-x86_64-linux
```

I like their automatic release notes generation:
```suggestion
          gh release create "v${TAG}" -t "v${TAG}" --generate-notes --latest artifacts/hermetic-driver-x86_64-linux
```